packages/waas/src/auth.ts

```diff
@@ -145,6 +127,34 @@ export function defaultArgsOrFail(
   return preconfig as Required<SequenceConfig> & Required<WaaSConfigKey> & ExtendedSequenceConfig
 }
 
+const fetch = globalThis.fetch
+
+const jwksDev = {
+  keys: [
+    {
+      alg: 'RS256',
+      e: 'AQAB',
+      kid: '9LkLZyHdNq1N2aeHMlC5jw',
+      kty: 'RSA',
+      n: 'qllUB_ERsOjbKx4SirGow4XDov05lQyhiF7Duo4sPkH9CwMN11OqhLuIqeIXPq0rPNIXGP99A7riXTcpRNk-5ZNL29zs-Xjj3idp7nZQZLIU1CBQErTcbxbwUYp8Q46k7lJXVlMmwoLQvQAgH8BZLuSe-Xk1tye0mDC-bHvmrMfqm2zmuWeDnZercU3Jg2iYwyPrjKWx7YSBSMTXTKPGndws4m3s3XIEpI2alLcLLWsPQk2UjIlux6I7vLwvjM_BgjFhYHqgg1tgZUPn_Xxt4wvhobF8UIacRVmGcuyYBnhRxKnBQhEClGSBVtnFYYBSvRjTgliOwf3DhFoXdnmyPQ',
+      use: 'sig'
+    }
+  ]
+}
```

Comment:
I feel like there should be a way to possibly override this key or disable it entirely. In the current form, it won't work locally or in any potential new environments we may create. So it'd be good to have an option to pass `null` to disable or a key to override.